### DISCUSSION TITLE
Keyboard command to toggle expand/collapse

### DISCUSSION THREAD

**Discussion by dkuep** [vincode-io/Zavala#113] [General]
Is it possible to define a keyboard command to toggle expand/collapse (the behaviour of cmd-0/9, but in one 'toggle' key)? e.g. Dynalist has this implemented with cmd-"."
Potentially this could be useful for "Collapse All in Row", "Collapse All in Outline" options. 

Thank you :)

**Comment by vincode-io** [comment]
I've been thinking a lot about this and _probably_ won't be implementing it. I want to keep Zavala as simple as possible and adding too many ways to do the same thing can get confusing. I do appreciate the suggestion though and haven't completely ruled out doing it.
